src/main/java/io/openshift/launchpad/BoosterCatalogFactory.java

```diff
@@ -66,6 +68,22 @@ public void reset()
       }
    }
 
+   public String[] getFilterLabels(UIContext context) {
+      Map<Object, Object> attributeMap = context.getAttributeMap();
+      List<String> labels = (List<String>) attributeMap.get(LABEL_FILTERS_PROPERTY_NAME);
```

Comment:
How about storing as a `String[]` and avoid the split and additional checks?

Reply:
That's just how it's getting passed from the UI to the rest backed.  See:

https://github.com/openshiftio/launchpad-backend/blob/master/src/main/java/io/openshift/launchpad/backend/rest/LaunchResource.java#L468

Reply:
Ah that makes sense